extensions/amp-analytics/0.1/vendors.js

```diff
@@ -349,7 +349,41 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
       'image': true,
     },
   },
-
+	
+'catchpoint': {
+	
+    'requests': {	
+      'endpoint': 'https://r.3gl.net/hawklogserver/r.p', 
+      'pageview': 'https://r.3gl.net/hawklogserver/r.p' +
```

Comment:
Just FYI: we will be encoding the data value here. The request will look something like `https://r.3gl.net/hawklogserver/r.p?data={%22v%22:2,%22y%22:%22load%22,%22ui%22:123324,%22.....`